misc/python/materialize/mzbuild.py

```diff
@@ -290,14 +290,21 @@ def inputs(self) -> set[str]:
             # *lot* of work.
             "Cargo.lock",
             ".cargo/config",
+            ".bazelrc",
+            "WORKSPACE",
+            # Even though we are not always building with Bazel, consider its
+            # inputs so that developers with CI_BAZEL_BUILD=0 can still
+            # download the images from Dockerhub
+            ".bazelrc",
+            "WORKSPACE",
```

Comment:
These files were listed twice?
```suggestion
            # Even though we are not always building with Bazel, consider its
            # inputs so that developers with CI_BAZEL_BUILD=0 can still
            # download the images from Dockerhub
            ".bazelrc",
            "WORKSPACE",
```